Week8/geg222/Mission2/src/components/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+const Footer = () => {
+  return (
+    <footer className="bg-black text-white text-sm mt-10 py-10 px-4 text-center border-t border-gray-800">
+      <div className="max-w-screen-lg mx-auto flex flex-col items-center gap-4">
+        <nav className="flex flex-wrap justify-center gap-4 font-semibold text-sm"></nav>
+
+        <div className="flex gap-6 mt-4 items-center justify-center">
+          <a
+            href="https://www.instagram.com/wonjunyyoung/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              src="/public/insta2.webp"
```

Comment:
```suggestion
              src="/insta2.webp"
```
public 폴더에 있는 파일들은 Vite에서 자동으로 루트 경로로 서빙되기 때문에, 위처럼 사용해야 정상적으로 불러와집니다!
여기 말고 다른 에셋들도 `/public/image.webp` 이런식으로 불러오고 있는 것 같은데 수정하면 좋겠습니다!
![image](https://github.com/user-attachments/assets/0aed711c-52da-41dc-aafc-f0dd9d02f154)

(참고: https://ko.vite.dev/guide/assets#the-public-directory)